server/query.js

```diff
@@ -83,15 +84,43 @@ const Query = {
     async getUser(req, res, id) {
         await query(req, res, `SELECT * FROM "User" WHERE id='${id}'`)
     },
-    async getCourses(req, res, id) {
-        await query(req, res, `SELECT * FROM course WHERE id IN (SELECT course_id FROM enrolledin WHERE student_id=${id})`)
+    async getCourses(req, res, id, userType) {
+        if (userType === "student") {
+            await query(req, res, `SELECT * FROM course WHERE id IN (SELECT course_id FROM enrolledin WHERE student_id=${id})`)
+        }
+        else if (userType === "staff") {
+            await query(req, res, `SELECT * FROM course WHERE id IN (SELECT course_id FROM teaches WHERE staff_id=${id})`)
+        }
     },
     async getAssignments(req, res, id) {
         await query(req, res, `SELECT * FROM assignment WHERE course_id=${id}`)
     },
     async unEnroll(req, res, uid, cid) {
         await query(req, res, `DELETE FROM enrolledin WHERE student_id=${uid} AND course_id=${cid}`)
     }, 
+
+    async deleteCourse(req, res, cid) {
+        await multiQuery(req, res, 
+            [`DELETE FROM teaches WHERE course_id=${cid}`,
```

Comment:
i think we can change the schemas to use `ON DELETE CASCADE` so that we can keep this as one query and not a bunch, but that can be done in a later branch